app/templates/gdt/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "dependencies": {
     "grunt": "^1.0.0",
-    "grunt-drupal-tasks": "~0.10.0",
+    "grunt-drupal-tasks": "^1.0.0-alpha",
```

Comment:
I believe it should.  But honestly this was just a placeholder so I could work with it since we needed the gdt composer support.  I'd much rather get GDT 1.0 pushed out there so that generator can just use "^1.0.0" or something.